src/views/blocks/tickets.php

```diff
@@ -28,9 +31,15 @@
 if ( ! $provider || empty( $tickets ) ) {
 	return false;
 }
-?>
 
-<?php $this->template( 'blocks/attendees/order-links' ); ?>
+$html = $this->template( 'blocks/attendees/order-links', [], false );
+
+if ( empty( $html) ) {
```

Comment:
:no_entry_sign: There must be a single space before the last parenthesis

<sub>hash: 01c864baf1fb4b44046990069d5d56e3</sub>